config/schema.yaml

```diff
@@ -667,11 +667,10 @@ schema:
         type: string
         doc: A string value composed only of letters, numbers, and underscores.
 
-  dataUnitRegions:
-    -
-      relates:
-        - Visit
-        - Sensor
+  dataUnitJoins:
+    VisitSensorRegion:
+      lhs: [Visit]
+      rhs: [Sensor]
```

Comment:
Just thinking aloud - maybe we should not call them left or right, they are totally interchangeable.

Reply:
Fair enough. Maybe we should just go back to `relates` for regular joins as well? The only place where it would make a difference is when you want to say `LEFT OUTER JOIN` or something in the SQL that defines the view and direction makes sense?

Reply:
OUTER JOIN direction is probably context-dependent so I'm not sure that is a good argument. And I'm not advocating changing it on this ticket, just an idea for future improvements.